ml/src/main/scala/org/apache/spark/ml/h2o/models/H2OMOJOModel.scala

```diff
@@ -125,9 +125,9 @@ class H2OMOJOModel(override val uid: String) extends H2OMOJOModelBase[H2OMOJOMod
           val pred = easyPredictModelWrapper.predictMultinomial(RowConverter.toH2ORowData(r))
           MultinomialPrediction(pred.classProbabilities)
         }
-        case ModelCategory.Clustering => udf[ClusteringPrediction, Row] { r: Row =>
+        case ModelCategory.Clustering => udf[ClusteringPredictionDetails, Row] { r: Row =>
           val pred = easyPredictModelWrapper.predictClustering(RowConverter.toH2ORowData(r))
-          ClusteringPrediction(pred.cluster)
+          ClusteringPredictionDetails(pred.cluster, pred.distances)
```

Comment:
As this is a new algorithm exposed, I wanted to make sure the `getPrediction` column contains directly the single value as it's in spark, more details are available in `getPredictionDetails` column